libs/model/src/models/thread.ts

```diff
@@ -140,12 +141,26 @@ export default (
             transaction: options.transaction,
           });
 
+          const { topic_id, community_id } = thread.get({
+            plain: true,
+          });
+          const contestManagers = (await GetThreadContestManagers().body({
+            actor: {} as Actor,
+            payload: {
+              topic_id,
+              community_id,
+            },
+          })) as { contest_address: string }[];
```

Comment:
For some reason, the return value is not reflecting the output type of the schema.

Reply:
@rbennettcw just pushed a commit with a fix for types... you can keep it if that works for you